README.md

```diff
@@ -24,9 +18,8 @@ For a more detailed view of how we think about Lyra, check out our [documentatio
 1. Clone the git repo from your $GOPATH: 
 `$ git clone https://github.com/lyraproj/lyra`
 
-3. Add the following lines to data.yaml
+2. Add the following lines to data.yaml
```

Comment:
Suggest we put these values into the manifest for now; Explaining how `lookup()` works is beyond the scope of "Hello, world!", imo. Also, it doesn't appear that `region`, `keyname`, or `instance.count` get used at all.

Reply:
Added